libchisel/src/verifyimports.rs

```diff
@@ -276,6 +276,7 @@ pub fn imported_func_sig_by_index(module: &Module, index: usize) -> FunctionType
     let import_section = module.import_section().expect("No function section found");
     let type_section = module.type_section().expect("No type section found");
 
+    assert!(index < import_section.entries().len());
```

Comment:
Can you place this into a separate commit and perhaps leave a note (not sure a note is actually needed).

Reply:
getting rid of this, actually. this is just leftover from bug hunting.